components/raftstore/src/store/fsm/peer.rs

```diff
@@ -2382,6 +2382,7 @@ where
                     res.applied_term,
                     &res.metrics,
                 );
+                self.on_check_peer_complete_apply_logs();
```

Comment:
should also checked after `on_applied_snapshot`

Reply:
It can be checked at the end of each loop on handling msgs.